.gitignore

```diff
@@ -56,6 +56,7 @@ _testmain.go
 # Log files
 *.log
 *.log.jsonl
+*.log.gz
```

Comment:
Unrelated, just necessary for when large log files roll over during development.